test/main.test.js

```diff
@@ -24,10 +24,10 @@ describe("getIndex()", function() {
 		throttled();
 
 		clock.tick(99);
-		expect(callback.callCount).to.be(0);
+		proclaim.equal(callback.callCount, 0);
```

Comment:
A little nit-picky but I'd probably use `strictEqual` in all of these assertions, so that it compares with `===` rather than `==`.